projector-agent/src/main/kotlin/org/jetbrains/projector/agent/AgentLauncher.kt

```diff
@@ -102,6 +104,26 @@ public object AgentLauncher {
     disconnectByIpMethod?.invoke(null, ip)
   }
 
+  @JvmStatic
+  @Suppress("deprecation")
```

Comment:
I would vote against using deprecated stuff... I see there are some suggestions on replacement in Observable class, have you considered them?